### PR TITLE
Add typescript types

### PR DESCRIPTION
**Pull request by mikaelkaron**
More or less the same as in https://github.com/vfile/vfile-rename/pull/1 (adding typescript types)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,32 +20,37 @@
...
-    "index.js"
+    "index.js",
+    "types/index.d.ts"
   ],
   "dependencies": {
     "is-buffer": "^2.0.0",
     "js-yaml": "^3.0.0"
   },
   "devDependencies": {
+    "dtslint": "^4.0.7",
```
Comment (ChristianMurphy) [inline package.json]:
```suggestion
    "dtslint": "^4.0.0",
```

File: package.json [hunk 2]
```diff
@@ -20,32 +20,37 @@
...
     "nyc": "^15.0.0",
     "prettier": "^2.0.0",
     "remark-cli": "^9.0.0",
     "remark-preset-wooorm": "^8.0.0",
     "tape": "^5.0.0",
     "to-vfile": "^6.0.0",
+    "vfile": "^4.2.1",
```
Comment (ChristianMurphy) [inline package.json]:
```suggestion
    "vfile": "^4.0.0",
```

**Review by ChristianMurphy — APPROVED** [review]
Thanks @mikaelkaron!

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
Thanks, [released](https://github.com/vfile/vfile-matter/releases/tag/2.1.0)!

**Pull request merged**